astroid/visitors.py

```diff
@@ -0,0 +1,90 @@
+import abc
```

Comment:
By the way, I don't know the exact details of this issue, but shouldn't we have the license header? Since this seems like a completely new file, probably the Copyright belongs to a different entity than Logilab, as we used until now, but I really don't know anything about licenses and whatnot. I'm asking since an user reporter a couple of days ago the fact that we do not have license headers for some of our files and apparently we need to be consistent with each new one.